src/vma/dev/rfs_mc.cpp

```diff
@@ -157,7 +157,6 @@ bool rfs_mc::prepare_flow_spec()
 bool rfs_mc::rx_dispatch_packet(mem_buf_desc_t* p_rx_wc_buf_desc, void* pv_fd_ready_array)
 {
 	// Dispatching: Notify new packet to all registered receivers
-	p_rx_wc_buf_desc->reset_ref_count();
```

Comment:
If we dont reset the ref count at this stage we have to ensure the value of the ref count is 0 while we put the buffer at the rx_pool.
Hence, we should call reset_ref_count() when we request buffers from the global buffer pool at _cq_mgr::request_more_buffers()_.